addons/ai/functions/fnc_waypointParadrop.sqf

```diff
@@ -55,12 +55,15 @@ waitUntil {
 
     // Check if the aircraft is close enough to the waypoint to begin paradropping
     // Distance is based on the current speed of aircraft and the number of passengers
-    private _passengers = {assignedVehicleRole _x select 0 == "cargo"} count crew _vehicle;
+    _passengers = {assignedVehicleRole _x select 0 == "cargo"} count crew _vehicle;
```

Comment:
```suggestion
    _passengers = {(assignedVehicleRole _x) select 0 == "cargo"} count crew _vehicle;
```
Personal preference to increase readability of this statement.

Reply:
I prefer not to have parentheses around unary commands. This line reads naturally from left-to-right either way IMO.

Reply:
agree with @CreepPork